tests/unit/Console/ApplicationTest.php

```
@@ -0,0 +1,84 @@
+    {
+        self::assertSame(
+            Application::FALLBACK_VERSION,
+            Application::resolveVersion('definitely/not-a-real-package-for-tests'),
+            "Unknown Composer packages must fall back to 'Application::FALLBACK_VERSION' so the CLI stays runnable.",
+        );
+    }
+
+    public function testResolveVersionReturnsComposerPrettyVersionWhenPackageIsInstalled(): void
+    {
+        self::assertNotEmpty(
+            Application::resolveVersion(Application::NAME),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\scaffold\Console\Application' in method 'testResolveVersionReturnsComposerPrettyVersionWhenPackageIsInstalled'. (undefined)

(StaticAccess)